db/seeds/development.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+user_list = [
+  {
+    name: "John",
+    surname: "Smith"
+  },
+  {
+    name: "Alan",
+    surname: "Watts"
+  },
+  {
+    name: "Hilary",
+    surname: "Shepherd"
+  },
+  {
+    name: "Jack",
+    surname: "Howard"
+  },
+  {
+    name: "John",
+    surname: "Doe"
+  }
+]
+
+def make_email(data)
+  "#{data[:name].downcase}.#{data[:surname].downcase}@example.com"
+end
+
+user_list.each do |user_item|
+  user = User.new(
+    user_item.merge(
+      {
+        password: "Password1234!",
+        email: make_email(user_item)
+      }
+    )
+  )
+
+  user.save
```

Comment:
```suggestion
  user.save!
```
Use a bang version so you will know if it fails